effects.cwt

```diff
@@ -2733,7 +2740,12 @@ alias[effect:set_allow_title_revokation] = bool
 # TODO: Figure out
 ## scope = title
 ### Crown laws
-alias[effect:set_allow_vice_royalties] = bool/tier
+alias[effect:set_allow_vice_royalties] = bool
+
+# TODO: Figure out
+## scope = title
+### Crown laws
+alias[effect:set_allow_vice_royalties] = tier
```

Comment:
Same here would something like ``alias[effect:set_allow_vice_royalties] = { bool title }`` be possible